README.md

```diff
@@ -130,6 +130,20 @@ on how to deploy the project on a live system.
 
 `./stage1 runserver localhost:3000`
 
+### Running the cron jobs
+
+Legit Info uses cron jobs to routinely fetch legislation data from Legiscan. The cron job can be run using the `weekly.sh` (for stage 1; development) shell script. However,
```

Comment:
Mention the user has to be in the right directory to run the cron script.

Reply:
@shubham599 had good idea of passing in working directory to the job. More testing needed.